install_config/redeploying_certificates.adoc

```diff
@@ -623,17 +623,67 @@ EOF
 oc create -f -
 ----
 +
-Then, run the following to remove the environment variables:
+Then, run the following command to remove the environment variables:
 +
 ----
 $ oc env dc/router OPENSHIFT_CA_DATA- OPENSHIFT_CERT_DATA- OPENSHIFT_KEY_DATA- OPENSHIFT_MASTER-
 ----
 
-. Delete the `router-certs` secret:
+. Obtain a certificate.
+* If you use an external Certificate Authority (CA) to sign your certificates,
+create a new certificate and provide it to {product-title} by following your
+internal processes.
+* If you use the internal {product-title} CA to sign certificates, run the
+following commands:
++
+[IMPORTANT]
+====
+The following commands generate a certificate that is internally signed. It will
+be trusted by only clients that trust the {product-title} CA.
+====
++
+----
+$ cd /root
+$ mkdir cert ; cd cert
+$ oc adm ca create-server-cert \
+    --signer-cert=/etc/origin/master/ca.crt \
+    --signer-key=/etc/origin/master/ca.key \
+    --signer-serial=/etc/origin/master/ca.serial.txt \
+    --hostnames='*.hostnames.for.the.certificate' \
+    --cert=router.crt \
+    --key=router.key \
+
+----
++
+These commands generates the following files:
++
+** A new certificate named *_router.crt_*.
+** A copy of the signing CA certificate chain, *_/etc/origin/master/ca.crt_*.
+This chain can contain more than one certificate if you use intermediate CAs.
+** A corresponding private key named *_router.key_*.
+
+. Create a new file that concatenates the generated certificates:
++
+----
+$ cat router.crt /etc/origin/master/ca.crt router.key > router.pem
+----
+
+. Back up the old certificate:
++
+----
+$ oc export secret router-certs > ~/old-router-certs-secret.yaml
+----
+
+. Create a new secret to hold the new certificate and key, and replace the 
+contents of the existing secret:
 +
 ----
-$ oc delete secret router-certs
+$ oc create secret generic router-certs tls.crt=router.pem tls.key=router.key \ <1>
```

Comment:
Here the command is old way but not correct,  the correct one should be like:
`oc create secret generic router-certs --from-file=tls.crt=custom-router.crt --from-file=tls.key=custom-router.key --type=kubernetes.io/tls -o json | oc replace -f -`

another new way is like:
`oc create secret tls tls-secret --cert=path/to/tls.cert --key=path/to/tls.key`

Reply:
@zhaozhanqi, thanks! Do you agree that the new way should go in 3.9 and later, and the old way should go in 3.5-3.7?

Reply:
hmm,I'm not sure about that. maybe UI team can help that. 
@xingxingxia  Could you help confirm this:

2:Do you agree that the new command should go in 3.9 and later?
3:Do you agree that this old way should go in 3.5-3.7?

Reply:
`oc create secret tls` and `oc secrets new` both exist in 3.3 ~ 3.10. Only since 3.9, `oc secrets new` is deprecated. It is OK to use `oc create secret tls` in the doc.

Reply:
Thank you @xingxingxia!  The bug was against 3.5, but I'll send the change back to 3.3.